stmt.go

```diff
@@ -38,15 +39,15 @@ func (s *stmt) NumInput() int {
 }
 
 func (s *stmt) Exec(args []driver.Value) (driver.Result, error) {
-	panic("Using ExecContext instead")
+	return nil, fmt.Errorf("use ExecContext instead")
 }
 
 func (s *stmt) ExecContext(ctx context.Context, args []driver.NamedValue) (driver.Result, error) {
 	return s.conn.ExecContext(ctx, s.query, args)
 }
 
 func (s *stmt) Query(args []driver.Value) (driver.Rows, error) {
-	panic("Using QueryContext instead")
+	return nil, fmt.Errorf("use QueryContext instead")
```

Comment:
Same here.

Reply:
This one is also not necessary: https://pkg.go.dev/database/sql/driver#Queryer